### PR TITLE
Find two ways registering people information page

### PR DESCRIPTION
**Pull request by stevehenshaw**
New page giving information about step 1 of finding family.

Hooked up the 'find out more' button from 'finding-two-ways' page

### CODE REVIEW

**Review by matt-lethargic — APPROVED** [review]

File: cypress/integration/FindTwoWaysRegisteringPage.feature [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Feature: Find Two Ways Registering Page
```
Comment (WillGibson) [inline cypress/integration/FindTwoWaysRegisteringPage.feature]:
```suggestion
Feature: Find Two Ways - Registering Missing People Page
```

File: src/views/find-two-ways.html [hunk 2]
```diff
@@ -49,13 +48,14 @@ <h2>2. Actively looking for relatives</h2>
...
 
         <div class="govuk-grid-row">
           <div class="govuk-grid-column-full">
-
               {{ redcrossButton({
-                              text: "Find out more"
+                              text: "Find out more",
+                              href: "/finding-registering",
+                              attributes: { id: "find-out-more-button" }
                           }) }}
```
Comment (WillGibson) [inline src/views/find-two-ways.html]: Just indenting...

```suggestion
                text: "Find out more",
                href: "/finding-registering",
                attributes: { id: "find-out-more-button" }
              }) }}
```

File: src/views/finding-registering.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      </div>
+
+      <br>
+
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+
+          {{ redcrossButton({
+          text: "Find out more"
+          }) }}
```
Comment (WillGibson) [inline src/views/finding-registering.html]:
```suggestion
          {{ redcrossButton({
            text: "Find out more"
          }) }}
```

File: src/views/finding-registering.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+          <br/>
+          {{ redcrossButton({
+          text: "skip to check eligibility",
+          href: "/check-eligibility",
+          classes: "redcross-button--secondary",
+          attributes: { id: "skip-to-check-button" }
+          }) }}
```
Comment (WillGibson) [inline src/views/finding-registering.html]:
```suggestion
          {{ redcrossButton({
            text: "skip to check eligibility",
            href: "/check-eligibility",
            classes: "redcross-button--secondary",
            attributes: { id: "skip-to-check-button" }
          }) }}
```

**Review by WillGibson — COMMENTED** [review]
Looking good, just minor stuff.

File: src/views/finding-registering.html [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+      </div>
+    </div>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        {{ govukBackLink({
+        text: "Back",
+        href: "/find-two-ways",
+        classes: "redcross-back-link"
+        }) }}
```
Comment (WillGibson) [inline src/views/finding-registering.html]:
```suggestion
        {{ govukBackLink({
          text: "Back",
          href: "/find-two-ways",
          classes: "redcross-back-link"
        }) }}
```

**Review by WillGibson — APPROVED** [review]
Spotted another indent thing, suggestion there, approving.

**Pull request merged**